OracleAnalytics/dockerfiles/buildDockerImage.sh

```diff
@@ -0,0 +1,141 @@
+#!/bin/bash
+# 
+# Copyright (c) 2022 Oracle and/or its affiliates.
+#
+# Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl.
+#
+# Author: steve.phillips@oracle.com
+# Description: script to build a Docker image for Oracle Analytics Server
+# 
+# DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS HEADER.
+
+usage() {
+cat << EOF
+
+Usage: buildDockerImage.sh -v [version] [-s] [-c] [-q]
+Builds a Docker Image for Oracle Business Intelligence.
+  
+Parameters:
+   -v: version to build. Required.
+       Choose one of: $(for i in */; do echo -n "${i%%/}  "; done)
+   -c: enables Docker image layer cache during build
+   -s: skips the MD5 check of packages
+   -q: squash resulting image
+
+LICENSE UPL 1.0
+
+Copyright (c) 2022 Oracle and/or its affiliates.
+
+EOF
+exit 0
+}
+
+# Validate packages
+checksumPackages() {
+  echo "Checking if required packages are present and valid..."
+  md5sum -c Checksum.md5
+  exit_code="$?"
+  if [ "$exit_code" -ne 0 ]; then
+    echo "MD5 for required packages to build this image did not match!"
+    echo "Make sure to download missing files in folder $VERSION. See *.download files for more information"
+    exit "$exit_code"
+  fi
+}
+
+if [ "$#" -eq 0 ]; then usage; fi
+
+# Parameters
+VERSION="12.2.1"
```

Comment:
This doesn't match your actual version of `6.4`.

Reply:
fixed the default version